crates/bevy_render/src/mesh/primitives/dim2.rs

```diff
@@ -77,6 +82,278 @@ impl From<CircleMeshBuilder> for Mesh {
     }
 }
 
+/// Specifies how to generate UV-mappings for [`CircularSector`] and [`CircularSegment`] shapes.
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub enum CircularMeshUvMode {
```

Comment:
This is a little odd as an enum with only one variant. I would like a comment here recording how it might be extended.